dialogflow-app.js

```diff
@@ -939,6 +964,22 @@ class DialogflowApp extends AssistantApp {
     return null;
   }
 
+  /**
+   * Get the current DialogFlow intent.
+   *
+   * @return {string} The intent name.
+   * @private
+   * @dialogflow
+   */
+  getIntentName_ () {
```

Comment:
Why use a separate private method? Why not contain all this in the getIntentName public method?

Reply:
I just replicated the method structure from `getIntent`. I'm fine with a single public method.

Reply:
Ah okay, yeah in this case let's go with a single public method. Thanks!